modules/post/windows/escalate/golden_ticket.rb

```diff
@@ -124,14 +130,26 @@ def run
 
     if ticket
       print_good('Golden Ticket Obtained!')
-      ticket_location = store_loot("golden.ticket",
-                                   "base64/kirbi",
+      kirbi_ticket = Base64.decode64(ticket)
+      kirbi_location = store_loot("golden.ticket",
+                                   "kirbi",
                                    session,
-                                   ticket,
+                                   kirbi_ticket,
                                    "#{domain}\\#{user}-golden_ticket.kirbi",
                                    "#{domain}\\#{user} Golden Ticket")
 
-      print_status("Ticket saved to #{ticket_location}")
+      print_status("Ticket saved to #{kirbi_location}")
+
+      # TODO: Convert ticket to ccache
```

Comment:
If this isn't a CCache file, then it shouldn't be stored with the `.ccache` extension. The kirbi data is already stored in the kirbi format, so this is redundant and should be removed.

Reply:
I agree. I'm hoping to get the conversion before marking this PR as ready for review but if I don't, I'll remove this.